extensions/adobe/experience/decisioning/option-detail.schema.json

```diff
@@ -16,19 +16,34 @@
     "option-detail-datatype": {
       "properties": {
         "xdm:id": {
-          "title": "Option Identifier",
+          "title": "Decision Option Identifier",
           "type": "string",
           "format": "uri-reference",
           "description": "A unique identifier for the decision option."
         },
+        "xdm:version": {
```

Comment:
Same here - should this be xdm:decisionVersion?

Reply:
Do we have a notion of eTag? This "version" refers to the entity referenced above? It could be called "xdm:entityVersion" or "something:etag".
decisionVersion would not be an ideal choices, because the thing that has the version is not the "decision" per se.

Reply:
I moved to a different property name "repo:etag". It is used elsewhere and is used in the same way we are using it here: to denote a particular snapshot in time for an entity/resource